### PR TITLE
File added: Assignment 7 & 8

### PR DESCRIPTION
**Pull request by pooja-nagare**

### CODE REVIEW

File: Pooja/HTML/Assignment7/home.css [hunk 1]
```diff
@@ -0,0 +1,161 @@
+<style>
+    html{
+        scroll-behavior: smooth;
+    }
+   .nav a:hover{
```
Comment (Priyankashelke2411) [inline Pooja/HTML/Assignment7/home.css]: space required after :

File: Pooja/HTML/Assignment7/home.css [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+       background-color:rgb(16, 16, 44);
```
Comment (Priyankashelke2411) [inline Pooja/HTML/Assignment7/home.css]: space required after :

File: Pooja/HTML/Assignment7/home.css [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+       }
+    .f2 {
+        width: 50%;
+        font-family: Arial, Helvetica, sans-serif;
+        margin-left:40px;
```
Comment (Priyankashelke2411) [inline Pooja/HTML/Assignment7/home.css]: check everywhere space required in style after :

File: Pooja/HTML/Assignment7/home.css [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+        margin-top:40px; 
+        color: blanchedalmond;
+        }
```
Comment (Priyankashelke2411) [inline Pooja/HTML/Assignment7/home.css]: extra line required after every class end statement

File: Pooja/HTML/Assignment7/home.css [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+        margin-right: 75px;
+    }
+    .ss{
+        margin-right: 128px;
+    }
+    .sk{
+        margin-right: 5%;
+    }
+    .btnsubmit{
+        color:crimson;
```
Comment (Priyankashelke2411) [inline Pooja/HTML/Assignment7/home.css]: space required after :

File: Pooja/HTML/Assignment7/home.css [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+        margin-right: 114px;
+    }
+    .lr{
+        margin-right: 75px;
+    }
+    .ss{
+        margin-right: 128px;
+    }
+    .sk{
+        margin-right: 5%;
```
Comment (Priyankashelke2411) [inline Pooja/HTML/Assignment7/home.css]: extra line required at end of class statements

**Review by Priyankashelke2411 — CHANGES_REQUESTED** [review]

**Review by pritikekan — APPROVED** [review]

File: Pooja/HTML/Assignment7/home.css [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+        margin-bottom: 60px;
+        margin-left: 700px;
+        background-color: black;
+        width:10%;
+    }
+    #sec1, #sec2, #sec3{
+        height: 200px;
+        padding-top: 60px;
+    }
+</style>
```
Comment (Pawan1005) [inline Pooja/HTML/Assignment7/home.css]: no need of style tag here

File: Pooja/HTML/Assignment7/home.html [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+                <div class="c">
+                    <label class="lc"><b>Comments:</b></label>
+                    <textarea name="" cols="30" rows="10"></textarea>
+                </div>
+                <button class="btnsubmit">Submit</button>
+            </form>
+        </div>
+    </section>
+</body>
+</html>
```
Comment (Pawan1005) [inline Pooja/HTML/Assignment7/home.html]: indentation missing

File: Pooja/HTML/Assignment8/index.html [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+            Industrial revolution, nature increasingly became seen as the part of reality
+            deprived from intentional intervention: it was hence considered as sacred by some traditions (Rousseau,
+            American transcendentalism) or a mere decorum for divine providence or human history
+            (Hegel, Marx). However, a vitalist vision of nature, closer to the presocratic one, got reborn at the same
+            time,
+            especially after Charles Darwin.
+        </p>
+    </div>
+</body>
+</html>
```
Comment (Pawan1005) [inline Pooja/HTML/Assignment8/index.html:65]: indentation missing

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Amit727619 — APPROVED** [review]

File: Pooja/HTML/Assignment7/academic.html [hunk 10]
```diff
@@ -0,0 +1,131 @@
...
+						<option>Networking</option>
+						<option>Management</option>
+						<option>Marketing</option>
+					</select>
+					<simpletext>You can select multiple fields.</simpletext>
+				</div>
+			</form>
+		</div>
+	</body>
+</html>
```
Comment (Pawan1005) [inline Pooja/HTML/Assignment7/academic.html:131]: check indentation

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**